students/piotrowski_stanislaw/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,14 @@
+# https://snakify.org/lessons/if_then_else_conditions/problems/knight_move/
+# piotrsta
+
+col1 = int(input())
+row1 = int(input())
+col2 = int(input())
+row2 = int(input())
+
+if abs(col1 - col2) == 1 and abs(row1 - row2) == 2:
```

Comment:
kilka raz liczysz to samo np. `abs(col1 - col2)`, więc możesz utworzyć zmienną dla tych wyników i możesz skrócić to do jednego `if` i `else`.